src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <div class="card__image">
+          <img
+           class="card__image-size"
+           src="images/imac.jpeg"
+           alt="mono"
+          >
+        </div>
+        <div class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+        <div class="card__number">Код товара: 195434</div>
+        <div class="card__stars-and-response">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__response">Отзывов: 5</div>
+        </div>
+        <div class="card__text-and-price">
+          <div class="card__text-price">Цена:</div>
+          <div class="card__number-price">69 999 грн</div>
+        </div>
+        <div class="card__button">
+          <a
+           href="#"
+           class="card__link"
+           data-qa="hover"
+          >Купить</a>
```

Comment:
```suggestion
          <a
            href="#"
            class="card__link"
            data-qa="hover"
          >
            Купить
          </a>
```

Reply:
Thank you, i did all what u sayd, but test failed, i don's see my falses